src/course-unit/CourseUnit.jsx

```diff
@@ -220,12 +220,13 @@ const CourseUnit = ({ courseId }) => {
               <IframePreviewLibraryXBlockChanges />
             </Layout.Element>
             <Layout.Element>
-              <Stack gap={3}>
+              <Stack gap={3} className="h-100">
```

Comment:
Could you please make this change a separate PR? That should be a very quick approval/merge.

Reply:
@brian-smith-tcril Sure thing! Here it is. https://github.com/openedx/frontend-app-authoring/pull/1852

Reply:
@brian-smith-tcril Sure thing! Here it is. https://github.com/openedx/frontend-app-authoring/pull/1852